views/default/contests.html

```diff
@@ -34,37 +34,33 @@
 
         function dateToTimeStamp(d) {
             return [d.getFullYear(), d.getMonth() + 1, d.getDate()].map(formatTimeStamp).join('-')
-                 + 'T'
-                 + [d.getHours(), d.getMinutes(), d.getSeconds()].map(formatTimeStamp).join(':')
-                 + '+05:30';
+                    + 'T'
+                    + [d.getHours(), d.getMinutes(), d.getSeconds()].map(formatTimeStamp).join(':')
+                    + '+05:30';
         }
 
-        /* Set Reminder in Google Calendar */
-        function setReminder(contestName, siteName, startTime, duration, contestLink) {
+        /* Authorize Google Calendar user*/
+        function handleAuthorize(callback) {
 
             // Your Client ID can be retrieved from your project in the Google
             // Developer Console, https://console.developers.google.com
-            var CLIENT_ID = '{{=current.calendar_token}}';
```

Comment:
calendar is the correct spelling. You need to change it in 0firstrun.py.sample .
And yes it is on prod and working because 0firstrun.py is gitignored and I copied the exact variable while deploying.
Thanks a lot for pointing this out.